bluemira/equilibria/flux_surfaces.py

```diff
@@ -123,6 +123,43 @@ def connection_length(self, eq):
         """
         return np.sum(self._dl(eq))
 
+    def poloidal_angle(self, eq, x_strike, z_strike, gamma):
```

Comment:
It strikes me that this calculation only makes sense for an `OpenFluxSurface`, therefore I think it should not live in the `FluxSurface` base class